src/pages/professions.page.tsx

```diff
@@ -0,0 +1,68 @@
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import { PageMetadata } from '@stefanprobst/next-page-metadata';
+import { Fragment, useEffect } from 'react';
+
+import { usePageTitleTemplate } from '@/app/metadata/use-page-title-template';
+import { useAppDispatch, useAppSelector } from '@/app/store';
+import { useGetProfessionsQuery } from '@/features/common/intavia-api.service';
+import { usePersonsSearchFilters } from '@/features/entities/use-persons-search-filters';
+import { Professions } from '@/features/professions/professions';
+import { ProfessionsPageHeader } from '@/features/professions/professions-page-header';
+import { LeafSizing } from '@/features/professions/professions-svg';
+import { Origin } from '@/features/visual-querying/Origin';
+import type { ProfessionConstraint } from '@/features/visual-querying/visualQuerying.slice';
+import {
+  addConstraint,
+  ConstraintType,
+  selectConstraints,
+} from '@/features/visual-querying/visualQuerying.slice';
+
+export default function ProfessionsPage(): JSX.Element | null {
+  const metadata = { title: 'Profession Hierarchy' };
+
+  const dispatch = useAppDispatch();
+  const titleTemplate = usePageTitleTemplate();
+
+  const searchFilters = usePersonsSearchFilters();
+  const { data, isLoading } = useGetProfessionsQuery(searchFilters);
+
+  const origin = new Origin();
+
+  // for testing only
```

Comment:
This should probably be removed before merging the PR. At the moment the constraint can be defined in the professions page, which is not the desired behavior.

Reply:
You're right. Side note: What is the idea for these standalone pages? Are they not just for testing?

Reply:
Thanks!